plugin/build_baron.go

```diff
@@ -47,45 +50,78 @@ func (bbp *BuildBaronPlugin) Configure(conf map[string]interface{}) error {
 		}
 		webhookConfigured := webHook.Endpoint != ""
 		if !webhookConfigured && proj.TicketCreateProject == "" {
-			return fmt.Errorf("ticket_create_project and taskAnnotationSettings.FileTicketWebHook endpoint cannot both be blank")
+			grip.Error(message.WrapError(err, message.Fields{
+				"message":      "ticket_create_project and taskAnnotationSettings.FileTicketWebHook endpoint cannot both be blank",
+				"project_name": projName,
+			}))
+			hasValidationError = true
 		}
 		if !webhookConfigured && len(proj.TicketSearchProjects) == 0 {
-			return fmt.Errorf("ticket_search_projects cannot be empty")
+			grip.Error(message.WrapError(err, message.Fields{
+				"message":      "ticket_search_projects cannot be empty",
+				"project_name": projName,
+			}))
+			hasValidationError = true
 		}
 		if proj.BFSuggestionServer != "" {
 			if _, err := url.Parse(proj.BFSuggestionServer); err != nil {
-				return errors.Wrapf(err, `Failed to parse bf_suggestion_server for project "%s"`, projName)
+				grip.Error(message.WrapError(err, message.Fields{
+					"message":      fmt.Sprintf(`Failed to parse bf_suggestion_server for project "%s"`, projName),
+					"project_name": projName,
+				}))
+				hasValidationError = true
 			}
 			if proj.BFSuggestionUsername == "" && proj.BFSuggestionPassword != "" {
-				return errors.Errorf(`Failed validating configuration for project "%s": `+
-					"bf_suggestion_password must be blank if bf_suggestion_username is blank", projName)
+				grip.Error(message.WrapError(err, message.Fields{
```

Comment:
message.WrapError isn't what you want in some of these cases; if err doesn't exist, this will no op.